Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -246,6 +263,28 @@ private extension Method {
         }
         return parts.compactMap { $0 }.joined(separator: " ")
     }
+
+    func functionName() -> String {
+        // Generics should now be added manually
+        "\(callName)(\(functionParameters()))"
+    }
+
+    func functionParameters() -> String {
+        parameters.map { parameter in
+            let typeName = parameter.typeName.wrapOptionalIfNeeded()
+            let argumentLabel = [parameter.argumentLabel, parameter.name]
```

Comment:
can you add some comments here, why this is needed? I really had to carefully read these lines to understand what's happening, but still don't know why it's needed